examples/openapi-tools/quickstart-se/README.md

```diff
@@ -0,0 +1,383 @@
+```bash
+mvn package
+java -jar target/openapi-java-server.jar
+```
+
+To check that server works as expected run the following `curl` commands :
+
+```
+curl -X GET http://localhost:8080/greet
+{"message":"World","greeting":"Hello"}
+
+curl -X GET http://localhost:8080/greet/Joe
+{"message":"Joe","greeting":"Hello"}
+
+curl -X PUT -H "Content-Type: application/json" -d '{"greeting" : "Hola", "message":"Lisa"}' http://localhost:8080/greet/greeting
+
+curl -X GET http://localhost:8080/greet
+{"message":"Lisa","greeting":"Hola"}
+```
+
+## Build, prepare and run the Helidon SE client
+
+The second part of this example is generating Helidon Webclient that will communicate with the server that we have just created.
+
+To generate Helidon SE Webclient at first we create `se-client` folder and then inside it we run the following command where `path-to-generator` is the directory where you downloaded the generator CLI JAR file and `path-to-openapi-doc` is the folder where `quickstart.yaml` is located:
+```bash
+java -jar path-to-generator/openapi-generator-cli.jar \
+          generate \
+          -g java-helidon-client \  
+          --library se \
+          -i path-to-openapi-doc/quickstart.yaml
+```
+
+When this command finishes its work in the folder `se-client` we will find the generated project. 
+As with the server project the most interesting parts are located inside `api` and `model` packages and `ApiClient` class.
+The package `api` contains interfaces that represent endpoints to our server and implementations for them.
+The package `model` contains classes that represent transport objects that will be used to communicate with the server.
+`ApiClient` class represents configuration and utility class for `WebClient` that is used to connect to our server.
+
+You can use the generated SE client artifact in either of two ways:
+
+ - as a library - One or more other client projects can depend on the client artifact and use its generated classes.
+ - as a client program itself - Add some code to the generated project to make it a client program and not just a library.
+
+This example illustrates the second approach. We create a second server (at port 8081) which accepts greeting requests and, acting as a client, forwards them to the first service and returns the responses from the first service as its own.
+
+To make our client application fully functional let's add some classes, dependencies and files to the project.
+
+1) Add  to the `pom.xml` :
+```xml
+    <properties>
+        <mainClass>org.openapitools.client.Main</mainClass>
+    </properties>
+
+    <dependency>
+        <groupId>io.helidon.webserver</groupId>
+        <artifactId>helidon-webserver</artifactId>
+    </dependency>
+    <dependency>
+        <groupId>io.helidon.config</groupId>
+        <artifactId>helidon-config-yaml</artifactId>
+    </dependency>
+```
+
+2) Let's add a class `MessageService` that will use `MessageApi` and `ApiClient` to interact with the server :
```

Comment:
We should either:

- add "to the `api` package" after "add a class `MessageService`", or
-  include the following line at the beginning of the code example: `package org.openapitools.client;` 

so it's clear where we want readers to add the file.